evasquez26/angular4-startup
# ONE-227 Clean sign up modal data

## Story / Task

see [ONE-227](lhttps://bixlabs.atlassian.net/browse/ONE-227)


## What this PR does?

Resolves bug related to the cleaning of the sign up modal.


### Before 

When you opened the sign up model, by clicking Sign Up, if a error message happened/ the form was submitted or you filled some field, closed the modal and then came back, messages and data were preserved. 

### After

Now when you click the sign up button in the header, the modal content is cleared.

### How to test / reproduce

- Run the app.
- Click 'Sign up' header button.
- Submit the form without filling all the fields.
- See that errors appear.
- Close the modal.
- Click 'Sign Up' again. You shouldn't be able to see old data/ old error messages.

### TODOs
None.

### Related PRs
N/A

## Migrations
N/A


------
Thanks!
